src/OpenLoco/Vehicles/Vehicle.h

```diff
@@ -422,15 +422,15 @@ namespace OpenLoco::Vehicles
     {
         static constexpr auto kVehicleThingType = VehicleThingType::vehicle_1;
         uint8_t pad_24[0x26 - 0x24];
-        EntityId head;              // 0x26
-        uint32_t remainingDistance; // 0x28
-        TrackAndDirection var_2C;
-        uint16_t subPosition;        // 0x2E
-        int16_t tileX;               // 0x30
-        int16_t tileY;               // 0x32
-        Map::SmallZ tileBaseZ;       // 0x34
-        uint8_t trackType;           // 0x35 field same in all vehicles
-        RoutingHandle routingHandle; // 0x36 field same in all vehicles
+        EntityId head;                       // 0x26
+        uint32_t remainingDistance;          // 0x28
+        TrackAndDirection trackAndDirection; // 0x2C
+        uint16_t subPosition;                // 0x2E
+        int16_t tileX;                       // 0x30
+        int16_t tileY;                       // 0x32
+        uint8_t tileBaseZ;                   // 0x34
```

Comment:
you've reverted the naming of the type of tileBaseZ to `Map::SmallZ`

Reply:
fuckin...I rebased to make sure I didn't revert these, not sure how this ended up like this :(